### PR TITLE
Restore will not work without the username

### PR DESCRIPTION
**Pull request by sean-horn**
Signed-off-by: Sean Horn <sean_horn@opscode.com>

### Description

Always asks for a password without the proper username specified.

We specify the supermarket username so that the pg_hba.conf allows us through with no password

### Issues Resolved

Unable to restore, asks for the password for a user we will not know

### Check List

- [ ] New functionality includes tests. Only docs
- [ ] All buildkite tests pass. Only docs
- [ ] Full omnibus build and tests in buildkite pass. Only docs
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/supermarket/supermarket_backup_restore.md [hunk 1]
```diff
@@ -95,5 +95,5 @@ The restore does not support transferring backups across different versions of S
 For example, to restore a backup in a .dump format, run:
 
 ```bash
-pg_restore --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket_production --verbose supermarket_database_backup.dump
+/opt/supermarket/embedded/bin/pg_restore -U supermarket --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket --verbose supermarket_database_backup.dump
```
Comment (RajeshPaul38) [inline docs-chef-io/content/supermarket/supermarket_backup_restore.md:98]: Hi @sean-horn ,
We have tested the changes you have suggested. It works fine but suggesting a small change from `-U` to `--username` just to maintain similarity for how we mentioned the other attributes for the pg_restore utility and giving the absolute path for the dump file to make sure this command runs from anywhere. Pls commit the suggestion if it looks good.
```suggestion
/opt/supermarket/embedded/bin/pg_restore --username supermarket --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket --verbose ~/supermarket_database_backup.dump
```

Comment (RajeshPaul38) [reply docs-chef-io/content/supermarket/supermarket_backup_restore.md:98]: Hi @sean-horn ,
Can you please work on this PR as per the review comments. Then we can close this.

**Review by RajeshPaul38 — APPROVED** [review]
LGTM

**Comment by RoyShravani** [comment]
Looks like we need to fix the DCO check failure.

**Comment by RajeshPaul38** [comment]
> PG restore command fixed in the docs #2783

@RoyShravani There is a separate PR that has been opened referring to this PR. Pls review this one: https://github.com/chef/supermarket/pull/2783

**Comment by RajeshPaul38** [comment]
Closing this as we have taken the inputs in this PR:  https://github.com/chef/supermarket/pull/2783

**Pull request closed without merging**